src/wp-includes/class-wp-embed.php

```diff
@@ -38,6 +39,7 @@ public function __construct() {
 		add_shortcode( 'embed', '__return_false' );
 
 		// Attempts to embed all URLs in a post.
+		add_filter( 'the_block_template_html', array( $this, 'autoembed' ), 6 );
```

Comment:
_If_ we go with my other suggestion on adjusting the shortcode filter priorities, we could align this:
```suggestion
		add_filter( 'the_block_template_html', array( $this, 'autoembed' ), 8 );
```